packages/bpk-component-slider/readme.md

```diff
@@ -0,0 +1,43 @@
+# bpk-component-slider
+
+> Backpack slider component.
+
+## Installation
+
+```sh
+npm install bpk-component-slider --save-dev
+```
+
+## Usage
+
+```js
+import React, { Component } from 'react';
+import BpkSlider from 'bpk-component-slider';
+
+const Slider = () => (
+  <BpkSlider
+    min={0}
+    max={100}
+    value={[20, 80]}
```

Comment:
What happens if `value` is given an array with more than two entries?

Reply:
It would have more than 3 handlers, I have not made CSS style for the ranges (horizontal bars in the slider) in case there are more than 3 (See picture). Should we make CSS for this situation or just avoid the user from using arrays with more than two entries?

<img width="517" alt="screen shot 2017-10-30 at 1 56 09 pm" src="https://user-images.githubusercontent.com/17517630/32174570-286c65d0-bd7a-11e7-8e12-814f2f12b6ea.png">

Reply:
Good question! I think it's probably best to give users the freedom to use more than two for any edge cases where that's necessary, in which case there should probably be some styles applied to the track between them. I think keeping it blue is fine though.

Reply:
I agree with @shaun - if it's easy to add and support is there then we should do it. However I wouldn't necessarily block this PR on this - it can be added at a later date/PR.